doc/decisions/0001-record-decisions.md

```diff
@@ -0,0 +1,40 @@
+# 1. Record Decisions
+
+| Date       | Tags    |
+|------------|---------|
+| 2018-02-16 | process |
+
+## Status
+
+Proposed
+
+## Context
```

Comment:
As soon as we have lots of them and it becomes a problem I'm happy to find solutions for this. At this time it seems like a non-issue to me.

Reply:
Why requiring extra work in the future if we could just create two folders right now? The problem is clear: non-devs don't mind about tech issues.

Reply:
Because your mapping "dev" / "non-dev" could turn out not to be appropriate and the amount of work to change this later is very small.

Reply:
@martinklepsch I agree with @pablodip - I think you should make two separate folders for organisational decisions and technical ones is a good idea to do now - have often seen small TODOs like this cause problems in `status-react` later on down the line. We can also have different templates for each, as the requirements are different.

Org Decision: Build a Admin Dashboard
Product Motivation:
Product Summary:
Goal:
Requirements:
Contributors:
Dates:

Technical Decision: Upgrade Web3j
Feature Motivation:
Feature Summary:
Goal:
Requirements:
Contributors:
Dates:

Or something along those lines, so that we can have both good overview and granular insight.

Reply:
@andytudhope I was hoping we could do this by using the tags in the document, maybe putting these into Markdown front-matter if required.

> have often seen small TODOs like this cause problems in status-react later on down the line.

Consider that the opposite can also be the case. Early decisions when we don't yet really know how we will be using a tool can be misinformed. What if a decision touches on dev and org? Tagging seems a more flexible approach but has not been implemented yet. 

Either way I feel like we can probably revert either decision if we feel like we have to. 

- `doc/decisions/org/`
- `doc/decisions/dev/`

Is this roughly what you're looking for?